pyiceberg/catalog/sql.py

```diff
@@ -329,8 +330,46 @@ def _commit_table(self, table_request: CommitTableRequest) -> CommitTableRespons
 
         Raises:
             NoSuchTableError: If a table with the given identifier does not exist.
+            CommitFailedException: If the commit failed.
         """
-        raise NotImplementedError
+        identifier_tuple = self.identifier_to_tuple_without_catalog(
+            tuple(table_request.identifier.namespace.root + [table_request.identifier.name])
+        )
+        current_table = self.load_table(identifier_tuple)
+        from_database_name, from_table_name = self.identifier_to_database_and_table(identifier_tuple, NoSuchTableError)
```

Comment:
To me, the `from_*` pattern, typically used alongside `to_*`, suggests a transformation, as we see in `rename_table` where the database and table name shift from A to B. But in `_commit_table`, where the names remain unchanged, I think sticking to `database_name` and `table_name` might be more straightforward. What do you think?

Reply:
Ah I missed that. Thank you for the suggestion! :)